conanfile.py

```diff
@@ -223,7 +223,7 @@ def configure(self):
             if self.settings.os == "Windows":
                 self.options["*"].shared = True
         print("cmake generator set to: " + statusColor + str(self.generator) + endColor)
-    
+
```

Comment:
Some rogue new formatting. Keeping formatting changes separate from functional changes is preferred.